homeassistant/components/squeezebox/entity.py

```diff
@@ -55,3 +55,12 @@ def __init__(
         self._attr_device_info = DeviceInfo(
             identifiers={(DOMAIN, coordinator.data[STATUS_QUERY_UUID])},
         )
+
+    @property
+    def available(self) -> bool:
+        """Return True if entity is available."""
+        # Check basic coordinator availability (i.e., last_update_success) and  coordinator.data is populated
+        if not super().available or not self.coordinator.data:
```

Comment:
why would coordinator.data not be populated? It either updated (last_update_success = True) or it failed (last update failed = False)